README.md

```diff
@@ -0,0 +1,151 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+=======
+
+## Capstone-Project: Programming Competion-page
+
+
+[Loom Video Presentation]()
+>>>>>>> efb449574a89f51f74a8b2fc2fbc8a9489b5f399
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
- [OPTIONAL ] Kindly remove commented readme file not used for this project to clean your README file and add some section which is missing from the original [Microverse temple](https://github.com/microverseinc/readme-template), like the featured section. :1st_place_medal: